src/controllers/playback/videoosd.js

```diff
@@ -1088,18 +1088,33 @@ define(["playbackManager", "dom", "inputManager", "datetime", "itemHelper", "med
         var NavigationKeys = ["ArrowLeft", "ArrowRight", "ArrowUp", "ArrowDown"];
 
         function onWindowKeyDown(e) {
+            var key = keyboardnavigation.getKeyName(e);
+
             if (!currentVisibleMenu && 32 === e.keyCode) {
                 playbackManager.playPause(currentPlayer);
                 showOsd();
                 return;
             }
 
-            if (layoutManager.tv && NavigationKeys.indexOf(e.key) != -1) {
+            if (layoutManager.tv && NavigationKeys.indexOf(key) != -1) {
                 showOsd();
                 return;
             }
 
-            switch (e.key) {
+            switch (key) {
+                case "Enter":
+                    showOsd();
+                    break;
+
```

Comment:
don't need a newline after breaks

Reply:
What about empty lines below? Also remove?
https://github.com/jellyfin/jellyfin-web/blob/ec6ce5aa14384f51ff4f97a749bb4e0985aed773/src/controllers/playback/videoosd.js#L1118-L1142
and so on

Reply:
sure